src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Extras/QuickGrid/BitQuickGridDemo.razor.scss

```
@@ -0,0 +1,131 @@
+.custom-grid {
+    border: 1px solid;
+
+    .container {
+        overflow: auto;
+    }
+
+    .flag {
+        vertical-align: middle;
+    }
+
+    ::deep {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)